README.md

```diff
@@ -289,6 +289,7 @@ git submodule deinit -f . && git submodule update --recursive --init
   - Make a copy of [sample.env](./sample.env) rename it as `.env` **_and_**;
   - Make a copy of [jpo-utils/sample.env](jpo-utils/sample.env), rename it as `.env` (keep this one in the `jpo-utils/` directory), and fill in the variables as described in the [jpo-utils README](jpo-utils/README.md)
   - **The .env files will contain private keys, do not add them to version control.**
+  - If you want to see log-based alerts notifying you if no TIMs were ingested in a specific period of time, you will want to update your `.env` file to set `ODE_TIM_INGEST_MONITORING_ENABLED=true` and `ODE_TIM_INGEST_MONITORING_INTERVAL=<seconds between ingest count checks>`. See [TimIngestWatcher](jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TimIngestWatcher.java) to see the log-based monitoring provided.
```

Comment:
Should this be updated to specify that the log-based TIM alerts are on by default?

Reply:
Good callout. I'll make the change!